tests/test_helpers_urls.py

```diff
@@ -0,0 +1,21 @@
+from masonite.helpers import urls_helpers
+
+
+def test_get_sets_route():
+    assert urls_helpers.get('test', None)
```

Comment:
In addition to all these asserts assert that they are instances of the correct HTTP classes. For instance something like:

```python
from masonite.routes import Get
assert isinstance(urls_helpers.get('test', None, Get)
```

that might not actually work right there because it might not be an actual instance. I forget how Python checks for instances but you get the point. Let me know if you have any questions about that. Do that for all the tests.

Reply:
@josephmancuso Check now please :)